app/src/main/java/org/p2p/wallet/home/ui/main/HomeFragment.kt

```diff
@@ -160,6 +160,25 @@ class HomeFragment :
         binding.viewActionButtons.showActionButtons(buttons)
     }
 
+    override fun showSwapWithArgs(tokenA: String, tokenB: String, amount: String, source: SwapOpenedFrom) {
```

Comment:
tokenA what? tokenA symbol, tokenA name?

Reply:
also in context of swap we can have amountA and amountB, it's better to specify that

Reply:
yep, there should be tokenASymbol, tokenBSymbol and amountA